### DISCUSSION TITLE
xstate/build package is coming ?

### DISCUSSION THREAD

**Discussion by iliasbhal** [statelyai/xstate#1523] [General] [answered]
@davidkpiano have you started working on `xstate/build`? 
if so I would love to be part of the work on the package. Are you open to it?
If not, would you consider a PR for that adds a `/build` package?

Note: I created the [xstate-builder](https://github.com/iliasbhal/xstate-builder) package that does exactly that a year ago.
With it, I can create pretty much any machine.
I would love to write a more polished version for `xtstate/build`!

Original Issue: https://github.com/davidkpiano/xstate/issues/880

**Comment by davidkpiano** [comment, marked as answer]
It's planned: https://github.com/davidkpiano/xstate/projects/1#card-40599055

Tentative API example here: https://gist.github.com/davidkpiano/ae330406e630c63fe22022cdceb36f32

I'll make a package for it in this repo, and would love help on it!

**Reply by mattpocock** [reply]
Yes! This looks great. A more functional API that can still output visualisable machines would be great for TS fans too. Package please 🥳
